doc/rst/tools/mason/mason.rst

```diff
@@ -750,9 +754,19 @@ Submit a Package
 The mason registry will hold the manifest files for packages submitted by developers.
 To contribute a package to the mason-registry a chapel developer will need to host their
 package and submit a pull request to the mason-registry with the toml file pointing
-to their package. For a more detailed description follow the steps below.
+to their package. For a more detailed description follow the steps below. Publishing
+can be done with ``mason publish`` or manually.
 
-Steps:
+``mason publish`` Steps:
+      1) Write a library or binary package in chapel using mason
+      2) Host the package in a git repository. (e.g. GitHub)
+      3) Fork the mason-registry on GitHub
+      4) Ensure your package has a remote origin.
+      5) Run ``mason publish`` in your package
+      6) Go to the link provided to open a pull request to the mason registry.
+      7) Wait for mason-registry gatekeepers to approve PR.
```

Comment:
Do we use gatekeepers elsewhere? If not, I'd suggest "maintainers" or "reviewers".

Reply:
Its used in the previous manual instructions for publishing a package so I figured I should just mirror it